tests/test_converters.py

```diff
@@ -95,3 +99,39 @@ def test_none_factory(self):
         c = default_if_none(default=Factory(list))
 
         assert [] == c(None)
+
+
+class TestChain(object):
+    def test_success(self):
+        """
+        Succeeds if all wrapped converters succeed.
+        """
+        c = chain(str, strtobool, bool)
+
+        assert True is c("True") is c(True)
+
+    def test_fail(self):
+        """
+        Fails if any wrapped converter fails.
+        """
+        c = chain(str, strtobool)
+
+        # First wrapped converter fails:
+        with pytest.raises(ValueError):
+            c(33)
+
+        # Last wrapped converter fails:
+        with pytest.raises(ValueError):
+            c("33")
+
+    def test_sugar(self):
+        """
+        `chain(c1, c2, c3)` and `[c1, c2, c3]` are equivalent.
+        """
+
+        @attr.s
+        class C(object):
+            a1 = attrib("a1", validator=chain(str, strtobool))
+            a2 = attrib("a2", validator=[str, strtobool])
```

Comment:
Yeah I was wondering why you're missing coverage and here we have it. :)
```suggestion
            a1 = attrib("a1", converter=chain(str, strtobool))
            a2 = attrib("a2", converter=[str, strtobool])
```

Reply:
My bad, Thanks!